2.0/Box/glTF-pbrSpecularGlossiness/Box.gltf

```diff
@@ -112,8 +112,26 @@
                     0.0,
                     0.0,
                     1.0
-                ]
-            }
+                ],
+                "metallicFactor": 0.0
+            },
+            "extensions": {
+                "KHR_materials_pbrSpecularGlossiness": {
+                    "diffuseFactor": [
+                        0.800000011920929,
+                        0.0,
+                        0.0,
+                        1.0
+                    ],
+                    "specularFactor": [
+                        0.20000000298023225,
```

Comment:
> I think it will be difficult for the validator to validate visual issues.

Sure. Implementing that (esp. software texture sampling) isn't on the roadmap at the moment. But such asset should be invalid, if we expect spec-defined relations between SG and MR.